src/components/views/messages/MImageBody.tsx

```diff
@@ -332,7 +332,12 @@ export default class MImageBody extends React.Component<IProps, IState> {
     _afterComponentWillUnmount() {
     }
```

Comment:
This is unused (one call, no use in subclasses), comment says MStickerBody uses it but it doesn't, can we kill it off as part of the TS conversion here. Not sure why the subclass wouldn't just have its own `componentWillUnmount` in which it calls` super.componentWillUnmount`